www/js/incident/post-trip-prompt.js

```diff
@@ -150,70 +140,18 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
-      if (data.identifier === 'REPORT') {
-          // alert("About to report");
+      if (data.identifier === 'MORE') {
+          Logger.log("Notification, action event");
           cleanDataIfNecessary(notification, state, data);
           displayCompletedTrip(notification, state, data);
-      } else if (data.identifier == 'SNOOZE') {
-        var now = new Date().getTime(),
-            _30_mins_from_now = new Date(now + 30 * 60 * 1000);
-        var after_30_mins_prompt = getTripEndReportNotification();
-        after_30_mins_prompt.at = _30_mins_from_now;
-        $window.cordova.plugins.notification.local.schedule([after_30_mins_prompt]);
-        if ($ionicPlatform.is('android')) {
-            $ionicPopup.alert({
-                title: "Snoozed reminder",
-                template: "Will reappear in 30 mins"
-            });
-        }
-      } else if (data.identifier === 'MUTE') {
-        var now = new Date().getTime(),
-            _1_min_from_now = new Date(now + 60 * 1000);
-        var notifyPlugin = $window.cordova.plugins.BEMTransitionNotification;
-        notifyPlugin.disableEventListener(notifyPlugin.TRIP_END, notification).then(function() {
-            if ($ionicPlatform.is('ios')) {
-                $window.cordova.plugins.notification.local.schedule([{
-                    id: REPORT,
-                    title: "Notifications for TRIP_END incident report muted",
-                    text: "Can be re-enabled from the Profile -> Developer Zone screen. Select to re-enable now, clear to ignore",
-                    at: _1_min_from_now,
-                    data: {redirectTo: "root.main.control"}
-                }]);
-            } else if ($ionicPlatform.is('android')) {
-                $ionicPopup.show({
-                    title: "Muted",
-                    template: "Notifications for TRIP_END incident report muted",
-                    buttons: [{
-                      text: 'Unmute',
-                      type: 'button-positive',
-                      onTap: function(e) {
-                        return true;
-                      }
-                    }, {
-                      text: 'Keep muted',
-                      type: 'button-positive',
-                      onTap: function(e) {
-                        return false;
-                      }
-                    }]
-                }).then(function(res) {
-                    if(res == true) {
-                        notifyPlugin.enableEventListener(notifyPlugin.TRIP_END, notification);
-                    } else {
-                        Logger.log("User chose to keep the transition muted");
-                    }
-                });
-            }
-        }).catch(function(error) {
-            $ionicPopup.alert({
-                title: "Error while muting notifications for trip end. Try again later.",
-                template: JSON.stringify(error)
-            });
-        });
+      } else if (data.identifier == 'BIKE') {
```

Comment:
what do you want to do if users pick `WALK` or `BIKE`? Currently, you do nothing. Is that what you want?

Reply:
I don't remember I've changed this. Might be an issue when doing merging. What should I do to this?

Reply:
@Andrew-Tan what *do* you want to do when the user chooses `'BIKE'` or `'WALK'` from the prompt at the end of the trip? Note that the default prompt says "Snooze" or "Mute". But the CCI group changed it to `'BIKE'` and `'WALK'` because they wanted to have one-click confirmation for the most common types of trips. You probably got this as part of the CCI merge.

You still want to get feedback on the trips, not just allow people to confirm mode. So maybe you don't want to change the buttons, or change them to something else?

You could go back to `'SNOOZE'` and `'MUTE'` and change it later based on the first level of feedback.

Current `'SNOOZE'`/`'MUTE'` functionality is in the master 
https://github.com/e-mission/e-mission-phone/blob/master/www/js/incident/post-trip-prompt.js#L23

Reply:
or you could just leave it as is for now. Just tell your initial users that the buttons don't work, and ask for feedback on what they should say.